BitwardenShared/UI/Auth/TwoFactorNotice/TwoFactorNoticeRoute.swift

```diff
@@ -0,0 +1,13 @@
+// MARK: - TwoFactorNoticeRoute
+
+/// A route to a specific screen in the No Two Factor notice.
+public enum TwoFactorNoticeRoute: Equatable, Hashable {
+    /// A route to dismiss the screen currently presented modally.
+    case dismiss
```

Comment:
⛏️ I would avoid specifying how the screen is presented so if then is not presented modally anymore we don't have to update the comment. Additionally, it's not like we have several `dismiss` cases for each type of presentation so we can simplify it.
```suggestion
    /// A route to dismiss the current screen.
    case dismiss
```

Reply:
I'll update it here, but there are several other instances of the comment elsewhere that should at some point be updated likewise 🤔  (I feel like that's a separate PR from this one, though)